dvl:
  ftdi: 7006fIP
  negate_x_vel: true
  negate_y_vel: false
  negate_z_vel: false